README.md

```diff
@@ -1,2 +1,13 @@
 # stylewithcss
-Guide to styling HTML with CSS
+
+## *Guide to styling HTML with CSS*
```

Comment:
I would skip the `*` emphasis here. Generally speaking, markdown should only be used to the effect of organizing and marking content, not styling it. Bold/italic should be left for specific elements where need to add additional emphasis, such as the use of `<strong>` and `<em>`.

```suggestion
## Guide to styling HTML with CSS
```